merchant-app/app/src/main/java/com/googleinterns/smb/ConfirmationActivity.java

```diff
@@ -0,0 +1,42 @@
+package com.googleinterns.smb;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import java.util.List;
+
+public class ConfirmationActivity extends AppCompatActivity {
+
+    private static final String TAG = "ConfirmationActivity";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_confirmation);
+        List<String> barcodes = (List<String>)getIntent().getSerializableExtra(ScanActivity.DETECTED_BARCODES);
+
+        ArrayAdapter<String> adapter = new ArrayAdapter<>(this,
+        android.R.layout.simple_list_item_1, barcodes);
+
+        ListView listView = findViewById(R.id.list_item);
+        listView.setAdapter(adapter);
+
+        // listening to single list item on click
+        listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+          public void onItemClick(AdapterView<?> parent, View view,
+              int position, long id) {
+
+              // selected item
+              String barcode = ((TextView) view).getText().toString();
+              Log.d(TAG, barcode);
+          }
+        });
```

Comment:
This seems a dead code from the UI view as no action and just logging is performed